### PR TITLE
Add README.md and update .gitignore

### PR DESCRIPTION
**Pull request by victorfern91**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configuration that we use at Feedzai in order check our code conventions and coding style in JavaScript projects. The objective of this package is provide the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
```
Comment (nancruz) [inline README.md]: I don't understand what you mean by: "in order check our code conventions and coding style in JavaScript projects."

Comment (nancruz) [inline README.md]: "The objective of this package is to provide the ESLint..."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+This repo includes two distinguished ESLint configurations:
```
Comment (nancruz) [inline README.md]: "This repo includes two distinguished (distinct) ESLint configurations:"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- `eslint-config-feedzai`- This package is our default configs that contain all of our ESLint rules, including our rules for ECMAScript 6+. If you're using React for your projects please check the `eslint-config-feedzai-react`.
```
Comment (nancruz) [inline README.md]: "This package is our default configs that contain all of our ESLint rules..." I think it could be rephrased to
"This package contains our ESLint rules, including the ones for ECMAScript 6+."

Comment (nancruz) [inline README.md]: "If you're using React for (in) your projects..."

File: README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+### Installing package dependencies
+
+In order to list the correct versions of each package you could use this command:
+
+```shell
+npm info "eslint-config-feedzai@latest" peerDependencies
+```
+
+After this command you could install those packages manually or if you're using the **npm version > 5** you could use this shortcut:
```
Comment (nancruz) [inline README.md]: "After this command(,) you could (can)..."

File: README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Add `"extends": "feedzai"` for JavaScript projects or `"extends": "feedzai-react"` for React projects to your `.eslintrc` file.
+
+## Development tasks
+
+You can check if those packages follow our coding style conventions by using `npm run lint` in the root folder of each package.
+
+## Deployment tasks
+
+**NOTE: ** Those tasks should be executed only in the  `master` branch.
```
Comment (nancruz) [inline README.md]: Only on the master branch? What happens if we want to publish a patch? The command should be run in a hf branch.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
```
Comment (lfac-pt) [inline README.md]: I would reword this as: "This repository contains the ESLint configuration used across products at Feedzai. We share this configuration to promote the use of linters in JavaScript and to share our conventions and best practices in the hope of inspiring others when defining their configurations."

File: README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+This repo includes two distinct ESLint configurations:
+
+- `eslint-config-feedzai`- This package contains our ESLint rules, including the ones for ECMAScript 6+.
```
Comment (lfac-pt) [inline README.md]: This configuration contains most of our rules, except the React related ones, including...

Calling configurations packages might be confusing.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
+
+This repo includes two distinct ESLint configurations:
+
+- `eslint-config-feedzai`- This package contains our ESLint rules, including the ones for ECMAScript 6+. 
+If you're using React in your projects please check the `eslint-config-feedzai-react`.
```
Comment (lfac-pt) [inline README.md]: Since the react config is two lines bellow this hardly seems necessary

File: README.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+npm install --save-dev eslint-config-feedzai-react eslint@^#.#.# eslint-config-feedzai@^#.#.# eslint-plugin-import@^#.#.# eslint-plugin-jest@^#.#.# eslint-plugin-jsx-a11y@^#.#.# eslint-plugin-react@^#.#.#
+```
+
+### Using Feedzai's ESLint configurations in your `.eslintrc` file
+
+Add `"extends": "feedzai"` for JavaScript projects or `"extends": "feedzai-react"` for React projects to your `.eslintrc` file.
+
+## Development tasks
+
+You can check if those packages follow our coding style conventions by using `npm run lint` in the root folder of each package.
```
Comment (lfac-pt) [inline README.md]: Does this repo actually contain two different packages? I'm confused

Comment (victorfern91) [reply README.md]: Yes, will include the `eslint-config-feedzai` and the `eslint-config-feedzai-react`.

File: README.md (hunk 8 above)
Comment (victorfern91) [reply README.md]: ok, I will remove that
